configuration/structures.go

```diff
@@ -103,6 +108,27 @@ type Bot struct {
 	WorkScheduler        WorkScheduler  `json:"work_scheduler,omitempty"`
 }
 
+type CreateBotRequestOptions struct {
+	Avatar               string         `json:"avatar_path,omitempty"`
+	DefaultGroupPriority GroupPriority  `json:"default_group_priority,omitempty"`
+	JobTitle             string         `json:"job_title,omitempty"`
+	MaxChatsCount        *uint          `json:"max_chats_count,omitempty"`
+	Groups               []*GroupConfig `json:"groups,omitempty"`
```

Comment:
```suggestion
	Groups               []GroupConfig `json:"groups,omitempty"`
```
Why is there a pointer to `GroupConfig`? Can it be `nil`?

Reply:
that was copied directly from `CreateBotRequest` in `configuration/api.go`. Will be fixed in the commit with post review fixes